packages/markdown/components/Anchor.jsx

```diff
@@ -0,0 +1,63 @@
+const React = require('react');
+const PropTypes = require('prop-types');
+
+// Nabbed from here:
+// https://github.com/readmeio/api-explorer/blob/0dedafcf71102feedaa4145040d3f57d79d95752/packages/api-explorer/src/lib/markdown/renderer.js#L52
+function getHref(href) {
+  const doc = href.match(/^doc:([-_a-zA-Z0-9#]*)$/);
+  if (doc) {
+    return `/docs/${doc[1]}`;
```

Comment:
Does all of this stuff break for enterprise stuff? Ie when the url should be `developers.intercom.io/building-apps/docs`

Reply:
That's a very good question... i've not changed this piece during the refactor, it came from the old markdown renderer: https://github.com/readmeio/api-explorer/blob/0dedafcf71102feedaa4145040d3f57d79d95752/packages/api-explorer/src/lib/markdown/renderer.js#L52

I'm not sure how this should be tackled really. Does this work in readme already? I wonder how we do it over there.

Reply:
Just tested this locally. The old explorer uses `ui-sref=docs.show({'doc': 'slug'})` which works to give the url `http://enterprise-group-parent.readme.local:3000/child/v1.0/docs/slug`. The new explorer gives a URL of `http://enterprise-group-parent.readme.local:3000/docs/slug`

Reply:
Wonder if we could give a basePath or something which it can use to construct all of these links?

Reply:
Hmm we'll probably have to do something like that, since I can't think of another way the api explorer will be aware it's in a child project. Annoying to have to pass another thing through though. I think since this is currently broken we should make it an additional task and not let it hold up this pr